spec2vec/Spec2VecParallel.py

```diff
@@ -4,14 +4,44 @@
 
 
 class Spec2VecParallel:
+    """Calculate spec2vec similarity scores between all references and queries.
 
+    Using a trained model, spectrum documents will be converted into spectrum
+    vectors. The spec2vec similarity is then the cosine similarity score between
+    two spectrum vectors.
+    """
     def __init__(self, model, intensity_weighting_power=0):
+        """
+
+        Parameters
+        ----------
+        model : gensim word2vec model
+            Expecgted input is a gensim word2vec model that has been trained on
+            the desired set of spectrum documents.
+        intensity_weighting_power : float, optional
+            Spectrum vectors are a weighted sum of the word vectors. The given
+            word intensities will be raised to the given power.
+            The default is 0, which means that no weighing will be done.
+        """
         self.model = model
         self.intensity_weighting_power = intensity_weighting_power
         self.vector_size = model.wv.vector_size
 
-    def __call__(self, references, queries):
+    def __call__(self, references, queries) -> numpy.array:
```

Comment:
Same as in Spec2Vec, you are using both typings and docstring types, choose one.

Reply:
Changed to typings.